### ISSUE TITLE
Go to module definition does not work from alias declaration

### ISSUE THREAD

**Issue by mirrec** [KronicDeth/intellij-elixir#1914]
**Describe the bug**
- CMD+click / "Go to declaration or usage" does not work from alias declaration
- Module aliasing in general works, because I can navigate to an aliased module from places where the aliased module is mentioned, or a function of an aliased module is called.

**To Reproduce**
1. create file `folder/page.ex` with module `defmodule Folder.Page do ... end`
2. alias this module in an other file `alias Folder.Page`
3. CMD+click on `Page` part in `alias Folder.Page`
4. Nothing happens, IDE does not navigate you to module `Folder.Page`

```elixir
defmodule ExampleWithAlias do
  alias Folder.Page # CMD+click / "go to Declaration or Usage" on `Page` does not work

  def hello do
    Page.fun_in_folder_page()
    # but it works from here ^
    # - when I click on `Page` I get list with 2 options
    #   - 1. the alias declaration
    #   - 2. the module
    # - when I click on `fun_in_folder_page`, I correctly get to the function declaration as expected
  end
end

```

**Expected behavior**
By clicking on a module in `alias Folder.Page`, you should be navigated to where module `Folder.Page` is defined

**Screenshots**

<img width="710" alt="go_to_definition_from_aliased_module_works" src="https://user-images.githubusercontent.com/544111/108919525-713a7c00-7633-11eb-94a7-5d4a3f7a509b.png">

**Desktop:**
 - OS: macOS
 - Version 10.15.7

**Erlang:**
- Installer: homebrew
- Version: Erlang/OTP 23 [erts-11.1.4] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe] [dtrace]

**Elixir:**
- Installer: homebrew
- Version: 1.11.2 (compiled with Erlang/OTP 23)

**Plugin:**
- Version: 11.10.0

**Additional context:**
- RubyMine 2020.3.2
- Build #RM-203.7148.67, built on January 26, 2021
- Runtime version: 11.0.9.1+11-b1145.77 x86_64
- VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
- macOS 10.15.7
- GC: ParNew, ConcurrentMarkSweep
- Memory: 1981M
- Cores: 8
- Non-Bundled Plugins: org.elixir_lang, NodeJS

**Comment by MTheProgrammer** [comment]
Same here, PHP Storm 2021. That's one of the most important feature for me in the JetBrains' IDEs

**Comment by JerzyDziala** [comment]
it's been like that for some time now in intellij idea in various versions

**Comment by KronicDeth** [comment]
This was fixed in 11.11.0

![Screen Shot 2021-06-28 at 1 40 52 PM](https://user-images.githubusercontent.com/298259/123687567-b1df1780-d816-11eb-9085-d9fa5181a568.png)
![Screen Shot 2021-06-28 at 1 41 50 PM](https://user-images.githubusercontent.com/298259/123687568-b1df1780-d816-11eb-86ab-fe498d5272a8.png)
![Screen Shot 2021-06-28 at 1 42 03 PM](https://user-images.githubusercontent.com/298259/123687569-b277ae00-d816-11eb-9b63-4e39d8759657.png)
